src/util/antlr/ANTLRErrorHandling.cpp

```diff
@@ -2,38 +2,76 @@
 //  Chair of Algorithms and Data Structures.
 //  Author: Julian Mundhahs (mundhahj@informatik.uni-freiburg.de)
 
-#include <util/antlr/ANTLRErrorHandling.h>
+#include "ANTLRErrorHandling.h"
```

Comment:
```suggestion
#include "util/antlr/ANTLRErrorHandling.h"
```

Always use absolute paths , even in the same directory (Does google do this otherwise, but I think it is more consistent)